.github/actions/e2e/slack/notify/action.yaml

```diff
@@ -30,9 +32,9 @@ runs:
       if: ${{ job.status == 'success' }}
       with:
         url: ${{ inputs.url }}
-        message: ":white_check_mark: ${{ steps.get-run-name.outputs.RUN_NAME }} (https://github.com/${{github.repository}}/actions/runs/${{github.run_id}})"
+        message: ":white_check_mark: ${{ steps.get-run-name.outputs.RUN_NAME }} (https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }})"
```

Comment:
sanity check: a run_id is generated by GH right? So there should be no way for this to be injected/modified by the user?

Reply:
Yes, that's correct. This is not user-generated so I chose to leave this as-is. Anything that was owned by Github and I know from looking over the contexts documentation cannot be user-generated, I left in the script: https://docs.github.com/en/actions/learn-github-actions/contexts